src/clj/ataru/forms/form_access_control.clj

```diff
@@ -100,6 +102,16 @@
          form-with-org
          :created-by (-> session :identity :username)))))))
 
+(defn edit-form-with-operations
+  [id operations session virkailija-tarjonta-service organization-service]
+  (if (empty? operations)
+    (throw (Exception. (str "Batching form " id " with no operations!"))))
```

Comment:
Need to throw? Maybe just do nothing or return some error message?

Reply:
Tuon vois kyllä poistaa. Frontti ei edes lähetä no-op patchejä